### ISSUE TITLE
Error: Data flow edge coming out of AssignSub node batch_normalization

### ISSUE THREAD

**Issue by maplesCM** [openvinotoolkit/openvino#329]
Hi,

I am using standard VGG model and followed by some batch normalization layers at the end. with this i am able to train , create .pb model & checkpoints. When i use this model on  "mo_tf.py --input_meta_graph ~/xxxx.meta --input_shape [1,x,x,3] " conversion getting an error like "Data flow edge coming out of AssignSub node batch_normalization ".

**Comment by ilyachur** [comment]
@maplesCM Can you try to use the last version of OpenVINO?

@lazarevevgeny FYI

**Comment by lazarevevgeny** [comment]
@maplesCM , when you provide a meta-graph as input to the Model Optimizer you need to specify the output node name. Since the meta-graph model contains some training operations with control dependencies the model conversion may fail with the error mentioned above.
